pages/linux/pivot_root.md

```diff
@@ -0,0 +1,13 @@
+# pivot_root
+
+> Change the root filesystem to a new directory and move the current root to another location.
+> Commonly used during system initialization (e.g., in initramfs) to switch from a temporary root to the real root filesystem.
+> More information: <https://manned.org/pivot_root.8>.
+
+- Move the current root to `/old_root` and make `/new_root` the new root:
+
+`sudo pivot_root /new_root /new_root/old_root`
```

Comment:
Can you give me an example on how this should look like in a real world scenario?